frontend/src/metabase/common/hooks/use-setting/use-setting.ts

```diff
@@ -1,12 +1,22 @@
 import { useCallback, useMemo } from "react";
 import _ from "underscore";
 
+// TODO - if I change this to metabase/admin/settings/selectors it borks.
```

Comment:
Probably some weird circular dependencies thing, `metabase/admin/settings/selectors` imports a lot and also exports some selectors that are used elsewhere.

I encountered that problem while working on this epic.

Reply:
I'll leave a comment and then we can deal with it once the other selectors are migrated